src/style.css

```
@@ -1 +1,116 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+body {
+  box-sizing: border-box;
+}
+
+form {
+ margin-top: 20px;
+ display: flex;
+ justify-content: space-between;
+ flex-direction: column;
+}
+
+.search-bar {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  position: relative;
+  border: 1px solid #E1E7ED;
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  border-radius: 4px;
+}
+
+.search-bar--big {
+  height: 70px;
+}
+
+.search-bar--small {
+  margin-top: 20px;
+  height: 42px;
+}
+
+.search-bar__icon-big {
+  margin-left: 26px;
+  height: 19px;
+  width: 20px;
+  border: 1px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 18px;
+  background-position: center;
+}
+
+.search-bar__icon-small {
+  margin-left: 13px;
+  height: 11px;
+  width: 11px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 10px;
+  background-position: center;
+}
```

```suggestion
.search-bar__icon {
  margin-left: 26px;
  height: 19px;
  width: 20px;
  border: 1px;
  background-image: url(images/Search.svg);
  background-repeat: no-repeat;
  background-size: contains;
  background-position: center;
}

.search-bar__icon--small {
  margin-left: 13px;
  height: 11px;
  width: 11px;
}
```
try to reduce amount of code by setting  modifiers for small form.
same for other elements.